### ISSUE TITLE
add GWAS viewer to Jbrowse

### ISSUE THREAD

**Issue by bauchetg** [solgenomics/sgn#1025]
See the add on here:
https://github.com/elsiklab/gwasviewer

**Comment by bauchetg** [comment]
@afpowell would you like to use this feature in SolGWAS?
A nice one is snp plotter, which simultaneously plot manhattan plot and LD:
https://cran.r-project.org/web/packages/snp.plotter/index.html
However I don t know if something similar exist in D3/Vega

**Comment by afpowell** [comment]
Yes, I think I should have something like this... it looks good! I'll make a note and check it out more later this week. Thanks!
